packages/synapse-react-client/test/components/SynapseTable/SynapseTable.integration.test.tsx

```diff
@@ -315,15 +317,17 @@ describe('SynapseTable tests', () => {
         direction: SORT_STATE[1],
       }
       // below we match only the part of the object that we expect to have changed
-      await waitFor(() =>
-        expect(executeQueryRequest).toHaveBeenCalledWith(
+      await waitFor(() => {
+        expect(executeQueryRequest).toHaveBeenCalled()
+        expect(typeof executeQueryRequest.mock.lastCall[0]).toBe('function')
+        expect(executeQueryRequest.mock.lastCall[0](lastQueryRequest)).toEqual(
```

Comment:
Many test changes account for the fact that `executeQueryRequest` is now called with a function